### DISCUSSION TITLE
RegisterPage.tsx:81 register -  AuthApiError: Database error saving new user

### DISCUSSION THREAD

**Discussion by nicitaacom** [supabase/supabase#16480] [Questions] [answered]
# What I'm doing
I'm creating auth logic rn and I want auth user after register and redirect him to '/'

### Error in dev tools console
RegisterPage.tsx:81 register -  AuthApiError: Database error saving new user
    at handleError2 (@supabase_supabase-js.js?v=a3b7f55d:4056:9)
    at async _handleRequest2 (@supabase_supabase-js.js?v=a3b7f55d:4091:5)
    at async _request (@supabase_supabase-js.js?v=a3b7f55d:4078:16)
    at async SupabaseAuthClient.signUp (@supabase_supabase-js.js?v=a3b7f55d:4620:15)
    at async register (RegisterPage.tsx:47:34)
    <hr/>
 ### Click register button on page localhost:8000/register to reproduce issue

  <hr/>

 ### I use React + Vite + TypeScript + tailwind + zustand + supabase
 
  <hr/>

 ### codesandbox - https://codesandbox.io/p/sandbox/admin-panel-zustand-jp43sq?file=%2Fsrc%2FApp.tsx%3A21%2C17

<hr/>

### path with issue -  src/pages/RegisterPage.tsx
 
<hr/>
 
 ### Also any suggestions will nice

**Comment by GaryAustin1** [comment]
Seems like it could be this.  https://github.com/orgs/supabase/discussions/13043   An insert trigger function on auth.users getting an error.

**Reply by nicitaacom** [reply]
How disable this trigger function?

Here is my error from logs
```
{"component":"api","error":"failed to close prepared statement: ERROR: current transaction is
 aborted, commands ignored until end of transaction block (SQLSTATE 25P02): 
ERROR: relation \"public.profiles\" does not exist (SQLSTATE 
42P01)","level":"error","method":"POST","msg":"500: Database error saving new 
user","path":"/signup","referer":"http://localhost:8001/","remote_addr":"32.33.33.333","time":"2023-08-
12T16:34:05Z","timestamp":"2023-08-12T16:34:04Z"}

```

**Reply by GaryAustin1** [reply, marked as answer]
So you added the user management starter (or something like it trigger function on new user insert).   But either did not create the profiles table or deleted it.  
`drop function my_trigger_function() cascade;`  in the SQL editor should drop both the function and the trigger.

**Comment by nicitaacom** [comment]
garyaustin
Thank you 
Solution was - go to the database - click functions - run your command in SQL edior with function name I want to delete
![image](https://github.com/supabase/supabase/assets/39565703/36b3fed6-4673-4a7d-83a9-af7537ce0cd4)

Also I think I could detele it my clicking ...
